packages/stream/evm/StreamClient.ts

```diff
@@ -176,13 +195,21 @@ export default class EvmStreamClient extends BaseStreamClient {
   }
 
   public async update(updateData: IUpdateData): Promise<ITransactionResult> {
-    console.log(updateData);
-    throw new Error("Not Implemented!");
+    const result = await this.writeContract.update(
+      updateData.id,
+      updateData.enableAutomaticWithdrawal ? [true] : [],
```

Comment:
Why is this an array?

Reply:
Limitation of the protocol, there are no Optional parameters in solidity, so a workaround is empty/non-empty array.